eos-retrigger-firstboot.service

```diff
@@ -0,0 +1,19 @@
+# On dual-boot systems, (re)run eos-firstboot to grow the root filesystem, which
+# was not grown correctly on first boot into Endless OS 6.0.0.
+[Unit]
+Description=Fix dual-boot filesystem size
+DefaultDependencies=no
+After=sysinit.target local-fs.target
+Before=basic.target eos-firstboot.service
+ConditionPathExists=!/var/%N
```

Comment:
I think Rob has previously expressed regret about dropping these files directly in `/var`. I.e., maybe this should be `/var/lib/%N`.

Reply:
> If we're going to unconditionally re-run eos-firstboot.service, why not just change the stamp file it uses?

Because I want to only re-run it on dual-boot systems.

But yes, that would be easier. It's harmless to rerun (except in terms of time wasted). It would also be an excuse to clean up the ugly top-level-of-/var stamp file. Nice idea.

Reply:
Ah, I missed the implications of `ConditionKernelCommandLine=endless.image.device`.

Reply:
I implemented that alternative at https://github.com/endlessm/eos-boot-helper/pull/410.